Classes/Player/AVPlayerEngine/AVPlayerEngine+AssetLoading.swift

```diff
@@ -48,6 +48,10 @@ extension AVPlayerEngine {
         if #available(iOS 10.0, tvOS 10.0, *) {
             playerItem.preferredForwardBufferDuration = newAsset.playerSettings.network.preferredForwardBufferDuration
         }
+        
+        if #available(iOS 11.0, *) {
```

Comment:
Please add the tvOS minimum version as well.
https://developer.apple.com/documentation/avfoundation/avplayeritem/2867324-preferredmaximumresolution